www/about/policies/terms-of-service.spt

```diff
@@ -238,17 +238,15 @@ responsible for the activity.</li>
 <li><p>Gratipay&#39;s Service Providers</p>
 
 <ol>
-<li>Gratipay uses Pound Payments Inc., DBA Balanced (&ldquo;Balanced&rdquo;) as a third
+<li>Gratipay uses Pound Payments Inc., Braintree (&ldquo;Braintree&rdquo;) as a third
```

Comment:
I believe Pound Payment is the same thing as Balanced based off of [this](https://poundpay.com/terms/marketplaceagreement) - I'm sure @whit537 has the most correct answer :)